pom.xml

```diff
@@ -75,9 +75,16 @@
             <version>${jakarta.json.version}</version>
             <scope>provided</scope>
         </dependency>
-        
+
         <!--  Tests -->
-		<dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
it seems that guava is used only to create Map for a test. I've found only one line occurrence.